test/Adapter/PhpCodeTest.php

```diff
@@ -26,111 +25,71 @@ public function setUp()
         $this->adapter = new Serializer\Adapter\PhpCode();
```

Comment:
Import the final class instead the only the namespace. This is the only class imported.